homework_4/script.js

```diff
@@ -0,0 +1,90 @@
+let page = 1;
+function loadArtworks() {
+  fetch('https://api.artic.edu/api/v1/artworks?page=${page}&limit=10')
```

Comment:
```suggestion
  fetch(`https://api.artic.edu/api/v1/artworks?page=${page}&limit=10`)
```

Super mi je kak cijelo vrijeme radiš dohvat istih slika jer si tu hardkodiro page parametar umjesto da je unutar template stringa 😂